OpenGL/Constructs/VAO.cs

```diff
@@ -618,7 +639,8 @@ private void DrawInstancedOGL2(int count)
         public void DrawProgram(ShaderProgram program)
         {
             BindAttributes(program);
-            Gl.DrawElements(DrawMode, VertexCount, elementType, IntPtr.Zero);
+            IntPtr offset = (IntPtr)(Offset * GetElementSizeInBytes());
+            Gl.DrawElements(DrawMode, VertexCount, elementType, offset);
```

Comment:
Similar to the comment above, it doesn't have to be for this commit, but I think it might be worth caching the computation of offset (especially since I imagine offset is usually zero) since this happens in the Draw code.